subprojects/docs/src/docs/userguide/javaPlugin.adoc

```diff
@@ -1005,17 +1005,17 @@ You can easily write a manifest to disk.
 How to upload your archives is described in <<artifact_management>>.
 
 [[sec:java_cross_compilation]]
-=== Compiling and testing for Java 6
+=== Compiling and testing for Java 6/7
 
-Gradle can only run on Java version 7 or higher. Gradle 3.x still supports compiling, testing, generating Javadoc and executing applications for Java 6. Java 5 is not supported.
+Support for running Gradle on Java 7 has been deprecated and is scheduled to be removed in Gradle 5.0. However, it still supports compiling, testing, generating Javadoc and executing applications for Java 6 and Java 7. Java 5 is not supported.
```

Comment:
:x: we should say "Gradle can only run on Java version 7 or higher. However, support for ..."

Reply:
Okay, so that's 

> Gradle can only run on Java version 7 or higher. However, support for running Gradle on
> Java 7 has been deprecated and is scheduled to be removed in Gradle 5.0. Gradle will 
> still supports compiling, testing, generating Javadoc and executing applications for Java
> 6. Java 5 is not supported.

Reply:
Done.